config/base/params.env

```diff
@@ -19,6 +19,7 @@ DSPO_HEALTHCHECK_DATABASE_CONNECTIONTIMEOUT=15s
 DSPO_HEALTHCHECK_OBJECTSTORE_CONNECTIONTIMEOUT=15s
 DSPO_REQUEUE_TIME=20s
 DSPO_APISERVER_INCLUDE_OWNERREFERENCE=true
+ARGOWORKFLOWSCONTROLLERS={"managementState":"Managed"}
```

Comment:
Could you please wrap the value of the environment variable in quotes to avoid parsing issues.

Reply:
done